src/base-element.js

```diff
@@ -586,40 +597,61 @@ export class BaseElement {
    * The handler is only invoked by events with trust equal to or greater than
    * `minTrust`. Otherwise, a user error is logged.
    *
-   * @param {string} method
+   * @param {string} alias
    * @param {function(!./service/action-impl.ActionInvocation)} handler
    * @param {ActionTrust} minTrust
    * @public
    */
-  registerAction(method, handler, minTrust = ActionTrust.HIGH) {
+  registerAction(alias, handler, minTrust = ActionTrust.HIGH) {
     this.initActionMap_();
-    this.actionMap_[method] = {handler, minTrust};
+    this.actionMap_[alias] = {handler, minTrust};
+  }
+
+  /**
+   * Registers the default action for this component.
+   * @param {function(!./service/action-impl.ActionInvocation)} handler
+   * @param {string=} alias
+   * @param {ActionTrust=} minTrust
+   * @public
+   */
+  registerDefaultAction(
+      handler, alias = DEFAULT_ACTION, minTrust = ActionTrust.HIGH) {
+    dev().assert(!this.defaultActionAlias_,
+        'There is already a default action registered.');
```

Comment:
For a more useful error message, print the already-registered alias name.

`dev().assert(!this.defaultActionAlias_, 'Default action "%s" already registered.', this.defaultActionAlias_);`